user/tutorials/oracle/oracle_with_multivariate_timeseries_data.ipynb

```diff
@@ -117994,34 +117982,23 @@
     "<a id='summary'></a>"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "159ecd06",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "In this tutorial, we have "
-   ]
-  },
   {
    "cell_type": "markdown",
    "id": "f0aebd95",
    "metadata": {},
    "source": [
     "In this tutorial, we have achieved the following:\n",
-    "1. We could understand what is H1st Oracle and how to build it from Rule-based Model (encoding expert knowledge) and unlabeled data. \n",
-    "2. We could evalute the performance of H1st Oracle and rule-based Model and found that Oracle outperforms the rule-based model even though we haven't used any labeled data to build Oracle. This is because Oracle includes discrimative models that can generalize the encoded rules of rule-based model and, furthermore, combine their intilligence through ensemble. \n",
-    "3. "
+    "1. We could understand what is H1st Oracle and how to build it from Rule-based Model (encoding expert knowledge) and unlabeled data. We also learned how to persist (save) the built Oracle and load (reuse) it. \n",
+    "2. We could evalute the performance of H1st Oracle and rule-based Model and found that Oracle outperforms the rule-based model even though we haven't used any labeled data to build Oracle. This is because Oracle includes discrimative models that can generalize the encoded rules of rule-based model and, furthermore, combine their intilligence through ensemble. "
```

Comment:
typos???
 `discrimative` => `discriminative`
`intilligence` => `intelligence`

Reply:
I got it. Let me update it.